### DISCUSSION TITLE
[FEATURE] Add TypeScript types

### DISCUSSION THREAD

**Discussion by RodrigoTomeES** [giorgiabosello/google-maps-react-markers#11] [Ideas]
**Is your feature request related to a problem? Please describe.**
When use this library in TypeScript it throw a lot of error because it doesn't have types.

**Describe the solution you'd like**
Add dts file with the types

**Describe alternatives you've considered**
Anyone

**Comment by giorgiabosello** [comment]
Should be fixed in version 1.0.9. Let me know :)

**Comment by RodrigoTomeES** [comment]
@giorgiabosello hi!, I tried the new version but the types error persists. I have seen that the dts files now exist with the types but it seems that something is wrong 🤔.

![Captura de pantalla 2022-12-03 a las 18 57 34](https://user-images.githubusercontent.com/13077343/205455061-c069ff69-acf6-4eed-8740-065595f97f7d.jpg)

It seems that the GoogleMap component is exported as type Any
![Captura de pantalla 2022-12-03 a las 18 58 13](https://user-images.githubusercontent.com/13077343/205455082-5313c839-e01c-400d-be0d-13634bba615a.jpg)

**Comment by giorgiabosello** [comment]
@RodrigoTomeES sorry, but there's no way to do this without writing the all library in typescript.

**Comment by omarkhatibgg** [comment]
@giorgiabosello, is it fine for you if I convert the library to Typescript and remove PropTypes?

**Reply by giorgiabosello** [reply]
Glad if you can :)
I'm currently studying typescript, so I'm not ready to that conversion by myself, yet.

**Reply by omarkhatibgg** [reply]
Sure, Happy to help!
I will probably open a PR this week

**Comment by giorgiabosello** [comment]
Hi guys, I completely rewritten the library in Typescript. Any feedback or help in improving it, is appreciated. 
Release is in alpha version, discussion here. #109 
Thanks 🙏🏼
